src/components/signup/InvalidWeb3TokenError.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { AsyncStorage, Image, StyleSheet, Text, View } from 'react-native'
+import normalize from '../../lib/utils/normalizeText'
+import { CustomButton, Section, Wrapper } from '../common'
+import Separator from '../common/layout/Separator'
+import Oops from '../../assets/oops.svg'
+import config from '../../config/config'
+
+const OutOfGasError = props => {
+  AsyncStorage.removeItem('web3Token')
```

Comment:
why removing again here?

Reply:
If user will try to go back to home page he will be redirected to recover again as there is functionality which checks if there is we3Token in AsyncStorage